libs/pika/thread_pools/include/pika/thread_pools/scheduling_loop.hpp

```diff
@@ -662,8 +662,8 @@ namespace pika::threads::detail {
                         num_thread, running, thrd, enable_stealing)))
             {
                 tfunc_time_wrapper tfunc_time_collector(idle_rate);
-                PIKA_ASSERT(get_thread_id_data(thrd)->get_scheduler_base() ==
-                    &scheduler);
+                //                PIKA_ASSERT(get_thread_id_data(thrd)->get_scheduler_base() ==
+                //                    &scheduler);
```

Comment:
This should probably be uncommented again, or we need to update _something_ (change the assert, add asserts elsewhere, etc.).

Reply:
I had to uncomment this to get the bypass scheduler to work, but I moved all the bypass scheduler stuff into another PR that is waiting, so I'll uncomment this for now